dist/screeps.d.ts

```diff
@@ -1017,8 +1017,7 @@ interface SignDefinition {
 }
 interface StoreDefinition {
     [resource: string]: number;
-    energy?: number;
-    power?: number;
+    energy: number;
```

Comment:
Typing something as:
``` interface StoreDefinition {
     [resource: string]: number;
     energy: number;
}```

Means that it is an object that maps string keys to numbers, but that all keys except energy may be absent (undefined).  This allows any resource type except energy to be undefined.
This is consistent with the manual (which states that "energy" is always present).

Perhaps we're miscommunicating: what would you say is the correct typing, then?